### DISCUSSION TITLE
Upgrade to 10.0.0 failing

### DISCUSSION THREAD

**Discussion by EugenMayer** [Netflix/dgs-framework#2093] [General]
Hello,

i have read the upgrade notes including https://netflix.github.io/dgs/announcements/#dgs-framework-1000-released-dec-17-2024but i'am seem to fail to understand the upgrade procedure.

Prior 10.0.0 (using 9.2.2) i was using those coordinates (catalog)

```
netflix-dgs = { module = "com.netflix.graphql.dgs:graphql-dgs", version.ref = "netflix-dgs"}
netflix-dgs-webmvc-autoconfig = { module = "com.netflix.graphql.dgs:graphql-dgs-spring-webmvc-autoconfigure", version.ref = "netflix-dgs"}
netflix-dgs-sb-autoconfig = { module = "com.netflix.graphql.dgs:graphql-dgs-spring-boot-oss-autoconfigure", version.ref = "netflix-dgs"}
netflix-dgs-extended-scalars = { module = "com.netflix.graphql.dgs:graphql-dgs-extended-scalars", version.ref = "netflix-dgs"}
```

So all of those have been used for our Netflix DGS projects (Projects based on SB 3.4.1).

With 10.0.0 i tried to follow the upgrade notes and only used

```
netflix-dgs = { module = "netflix.graphql.dgs:dgs-starter", version.ref = "netflix-dgs"}
netflix-dgs-extended-scalars = { module = "com.netflix.graphql.dgs:graphql-dgs-extended-scalars", version.ref = "netflix-dgs"}
```

In the end, i end up having not included any dependencies.

I currently want to use DGS only, not mixing both or switching the Spring GQL. 

Thank you for any hints

**Comment by hantsy** [comment]
According to the Dgs 10.0.0 release notes, it was already switched to Spring GraphQL as the execution layer. Dgs provides alternative annotations and some 3rd party extension integrations.

You have to include a `web` or `webflux` starter in the dependencies. 

I have just updated my example projects to 10.0.1, https://github.com/hantsy/spring-graphql-sample/tree/master

**Comment by EugenMayer** [comment]
So you still include the bom

https://github.com/hantsy/spring-graphql-sample/blob/master/dgs-kotlin/build.gradle.kts#L38C1-L40C6

and then dgs-started and extended scalars.

Of course i also include `"org.springframework.boot:spring-boot-starter-web"` in my case too (whis is what you refered to with web or webflux right?

**Comment by EugenMayer** [comment]
In the end it turned out to be a layer8 issue of the stupid kind - how ever i managed to do this, as you see above i used `netflix.graphql.dgs:dgs-starter` and not `com.netflix.graphql.dgs:dgs-starter`

Sorry for the nonsense question! :)

**Reply by paulbakker** [reply]
Thanks for reporting back @EugenMayer, glad you figured it out already.

**Reply by setchy** [reply]
@paulbakker - the [10.0.0 release notes](https://github.com/Netflix/dgs-framework/releases/tag/v10.0.0) may need a slight tweak from `netflix.graphql.dgs:` to `com.netflix.graphql.dgs:` 😄

**Reply by EugenMayer** [reply]
Oh here we go! I fell for the trap! :dancers:
